### ISSUE TITLE
Install-Module for PSReadLine 2.1.0 Beta 1 on PowerShell 7

### ISSUE THREAD

**Issue by aavdberg** [PowerShell/PowerShellGetv2#596]
When i try to install PSReadLine 2.1.0 Beta 1 on PowerShell 7 i get the following error:

![image](https://user-images.githubusercontent.com/22030609/78702403-d0eb4400-7908-11ea-9487-45c007e213be.png)

This is the information:

![image](https://user-images.githubusercontent.com/22030609/78702721-52db6d00-7909-11ea-831d-268cae7d2644.png)

**Comment by aavdberg** [comment]
And there is also no solution so getting nowhere only redirected from one place to the other.

**Comment by alerickson** [comment]
@aavdberg I was waiting for a response to Justin's comment above.  Are you running as admin?

**Comment by aavdberg** [comment]
Yes i am running as Admin, i solved the problem when i did see that Windows 10 Protected folders was blocking it.

![image](https://user-images.githubusercontent.com/22030609/79099809-41f58600-7d65-11ea-8f67-ea63bd6acb87.png)

![image](https://user-images.githubusercontent.com/22030609/79099914-79fcc900-7d65-11ea-938b-525934030bdd.png)

**Comment by aavdberg** [comment]
So did this command in cmd with administrator rights.

![image](https://user-images.githubusercontent.com/22030609/79100047-c3e5af00-7d65-11ea-969f-41b9974b4991.png)

**Comment by aavdberg** [comment]
But why powershell 7 gets blocked on Windows 10, where i have to report this and who will do something with it?

**Comment by alerickson** [comment]
For that, please open an issue in the [PowerShell](https://GitHub.com/PowerShell/Powershell) repo.

**Comment by alerickson** [comment]
For that, please open an issue on the [PowerShell](https://GitHub.com/PowerShell/PowerShell) repo.

**Comment by daxian-dbw** [comment]
Oh, that's a change in Windows that moves the user's Documents folder to OneDrive. That means the same issue will potentially happen to all modules that are ships with PowerShell, including PowerShellGet. @joeyaiello Can you please take a look here?

**Comment by aavdberg** [comment]
@daxian-dbw @joeyaiello have now the same with installing PowerShellGet 

Install-Module -Name PowerShellGet -AllowPrerelease -Force
![image](https://user-images.githubusercontent.com/22030609/84077455-a5074000-a9d7-11ea-9fae-9b76c3de18a6.png)

**Comment by daxian-dbw** [comment]
From @aavdberg:
> But why powershell 7 gets blocked on Windows 10, where i have to report this and who will do something with it?

@SydneyhSmith For context, Windows moved the personal document folder to OneDrive, and it looks it blocks PowerShell from accessing the folders there, including the user module path folder (see https://github.com/PowerShell/PowerShellGetv2/issues/596#issuecomment-612780010). Can you please point @aavdberg to the right place to report this issue?

**Comment by alerickson** [comment]
@aavdberg for the time being if you want a workaround you can update $env:PSModulePath to have the old documents folder prioritized on the path:
`
$env:PSModulePath = "<PreviousDocumentsPath>; $env:PSModulePath"`

**Comment by aavdberg** [comment]
Thank you @alerickson just Friday received my new Surface Book 3 and was setting up everything. So I don't have a previous path.

But I can create a new location and copy all the content from the current location to this new location when I understand right?

**Comment by alerickson** [comment]
@aavdberg, yup that'll work... the previous default documents path was:
 `C:\Users\<username>\Documents\PowerShell\Modules`

**Comment by daxian-dbw** [comment]
I doubt if that will work for `Install-Module`. If `Install-Module` still treats the onedrive path as the personal module path, then it will not try to install modules to the newly created location.

**Comment by aavdberg** [comment]
Indeed not working and also when i remove the onedrive location from the env.

**Comment by alerickson** [comment]
Ah okay, in that case as a workaround you can use `Save-Module -Destination <PreferredPath>`
This will install to the path specified.

**Comment by cygnim** [comment]
I also ran into this exact problem and can confirm saving the module to another location and then copying it to the local module path worked. Hopefully this can be fixed soon as the problem is not obvious.
